spyder/plugins/debugger/panels/debuggerpanel.py

```diff
@@ -87,11 +88,11 @@ def paintEvent(self, event):
 
         Paint breakpoints icons.
         """
-        super(DebuggerPanel, self).paintEvent(event)
         painter = QPainter(self)
         painter.fillRect(event.rect(), self.editor.sideareas_color)
 
         for top, line_number, block in self.editor.visible_blocks:
+            self.paint_cell(painter)
```

Comment:
Should this be call outside the `for` statement? Or maybe the logic inside should handle the case of passing a single block variable as argument? Otherwise I think we will be painting the line cell for all the blocks multiple times since calling this method here is doing a `for` inside a `for`, right?